core/contracts/financial-templates/expiring-multiparty/ExpiringMultiPartyCreator.sol

```diff
@@ -155,5 +104,7 @@ contract ExpiringMultiPartyCreator is ContractCreator, Testable, Lockable {
         constructorParams.sponsorDisputeRewardPct = params.sponsorDisputeRewardPct;
         constructorParams.disputerDisputeRewardPct = params.disputerDisputeRewardPct;
         constructorParams.minSponsorTokens = params.minSponsorTokens;
+        constructorParams.withdrawalLiveness = params.withdrawalLiveness;
```

Comment:
With this change, there is no validation done on either of these params at all; not even a check to ensure that theses values are `!=0`. This is fine I guess but would we not want to impose _some_ kind of bounds on these?

Reply:
No, you’re 100% correct. We should validate that they’re > 0. That’s an oversight. Will fix!

Do you think a >0 check is sufficient — just to make sure the liveness values force the caller to wait at least 1 block?

Reply:
I'll make this change.

Reply:
Done. PTAL.

Reply:
LGTM! `Liquidation liveness cannot be 0` is _exactly_ 32 chars so we right on the limit here